src/Admin/Utilities/RolePermissionMapping.cs

```diff
@@ -0,0 +1,180 @@
+﻿using Bit.Admin.Enums;
+
+namespace Bit.Admin.Utilities;
+
+public static class RolePermissionMapping
+{
+    public static readonly Dictionary<string, List<Permission>> RolePermissions = new Dictionary<string, List<Permission>>()
```

Comment:
Definitely looking forward to this being in a table!

Reply:
suggestion: a TODO here might be nice to mark this as temporary.